alf/examples/ppo_babyai.py

```diff
@@ -0,0 +1,140 @@
+# Copyright (c) 2021 Horizon Robotics and ALF Contributors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from functools import partial
+import torch
+
+import alf
+from alf.algorithms.agent import Agent
+from alf.algorithms.encoding_algorithm import EncodingAlgorithm
+from alf.algorithms.ppo_algorithm import PPOAlgorithm, PPOLoss
+from alf.algorithms.trac_algorithm import TracAlgorithm
+from alf.environments import suite_babyai, alf_wrappers
+
+alf.config(
+    'TrainerConfig',
+    algorithm_ctor=Agent,
+    unroll_length=64,
+    mini_batch_size=32,
+    num_iterations=50000,
+    num_updates_per_train_iter=2,
+    eval_interval=1000,
+    debug_summaries=True,
+    summarize_grads_and_vars=True,
+    summary_interval=100,
+    use_rollout_state=True,
+)
+
+alf.config(
+    'suite_babyai.load',
+    mode='sent',
+    alf_env_wrappers=[alf_wrappers.ActionObservationWrapper])
+
+alf.config(
+    'create_environment',
+    env_load_fn=suite_babyai.load,
+    env_name=[
+        "BabyAI-GoToObj-v0", "BabyAI-GoToRedBallGrey-v0",
+        "BabyAI-GoToRedBall-v0", "BabyAI-GoToLocal-v0", "BabyAI-PickupLoc-v0"
+    ],
+    num_parallel_environments=64,
+    batched_wrappers=(alf_wrappers.CurriculumWrapper, ),
+)
+
+observation_spec = alf.get_observation_spec()
+action_spec = alf.get_action_spec()
+mission_spec = observation_spec['observation']['mission']
+vocab_size = mission_spec.maximum + 1
+
+encoding_dim = 128
+activation = torch.relu_
+fc_layers_params = (256, 256)
```

Comment:
Will these global variables won't interfere with variables of the same names in python code?